CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 ### StripeApplePay
 * Fixed a compile-time issue with using StripeApplePay in an App Extension. ([#2853](https://github.com/stripe/stripe-ios/issues/2853))
 
+## 23.15.0 2023-08-22
```

Comment:
This line can be removed. The version number is computed at the time we deploy, see the first line which is the placeholder.